core/src/main/java/org/gbif/pipelines/core/ws/client/internal/GBIFInternalService.java

```diff
@@ -0,0 +1,25 @@
+package org.gbif.pipelines.core.ws.client.internal;
+
+import com.google.gson.JsonElement;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+
+/**
+ * retro client for gbif services.
+ */
+interface GBIFInternalService {
+
+  @GET("dataset/{datasetUUID}/networks")
+  Call<JsonElement> getNetworkFromDataset(@Path("datasetUUID") String UUID);
```

Comment:
getDatasetNetworks/ getNetworksByDataset or just getNetworks since we already have dataset key param